lib/python/qmk/cli/list_keyboards.py

```diff
@@ -0,0 +1,30 @@
+"""List the keyboards currently defined within QMK
+"""
+from milc import cli
```

Comment:
Following PEP8 this should be put after the system modules, with a blank line separating the system modules from 3rd party modules.